packages/communication-react/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "types": "dist/communication-react.d.ts",
   "dependencies": {
-    "@azure/communication-calling": "1.0.1-beta.1",
+    "@azure/communication-calling": "1.1.0-beta.1",
```

Comment:
Do we have best practices for when we increment versions?

The next version that we release should be called `1.0.0-beta.2`
@JamesBurnside

Reply:
This isn't bumping our version, but the SDK dependency.

But  +1, this PR isn't a good place to change dependencies because of the extra risk of breakage.